src/qibojit/backends/cpu.py

```diff
@@ -236,12 +236,19 @@ def _apply_ygate_density_matrix(self, gate, state, nqubits):
 
     def apply_channel_density_matrix(self, channel, state, nqubits):
         state = self.cast(state)
+        if channel.name == "ReadoutErrorChannel":
```

Comment:
This is fine, but maybe `isinstance(channel, ReadoutErrorChannel)` would be more robust as the class name changes harder than the `name` attribute. For example the user may do
```py
channel = ReadoutErrorChannel(...)
channel.name = "my_channel"
```
and break this, while the class name does not change as easily. Also at some point we may use the name for other purposes (eg. easier definition of noise models).

Reply:
Thanks. Changed it.